### ISSUE TITLE
Cannot get the accuracy reported on the paper

### ISSUE THREAD

**Issue by fanglinpu** [yuxiaochen1103/DG-STA#1]
I have run the code on the DHG-14 dataset, the avererage accuracy across all participents is only 89.9%, but the reported accuracy on the paper is 91.9%, what's the problem?

**Comment by yuxiaochen1103** [comment]
Hi, Thanks for your interests. We applied two residual connections when training on the DHG-14 dataset, and we will release the code and pre-trained models soon. You also can implement it by adding the input of spatial / temporal attention(128D-vector) with output of ReLu layer (128D-vector) after the correspondent attention model.

**Comment by fanglinpu** [comment]
Did you use the best validation accuracy during training for final evaluation?

**Comment by yuxiaochen1103** [comment]
I follow the same way as baseline methods to evaluate my model.

**Comment by Fazlik995** [comment]
@yuxiaochen1103 can you tell, please, what is baseline method/paper?
